settings.gradle

```diff
@@ -47,10 +45,15 @@ project(':arrow').name = 'iceberg-arrow'
 project(':parquet').name = 'iceberg-parquet'
 project(':bundled-guava').name = 'iceberg-bundled-guava'
 project(':spark').name = 'iceberg-spark'
-project(':spark2').name = 'iceberg-spark2'
-project(':spark-runtime').name = 'iceberg-spark-runtime'
 project(':spark3').name = 'iceberg-spark3'
 project(':spark3-runtime').name = 'iceberg-spark3-runtime'
 project(':pig').name = 'iceberg-pig'
 project(':hive').name = 'iceberg-hive'
 
+if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
```

Comment:
Does this mean that we do not publish spark2 artifacts when building using Java 11?

Reply:
Release builds are built with Java 8 and should run fine in Java 11. We just can't run the Spark 2.x tests in Java 11 because those versions of Spark are incompatible.

Reply:
Thks. That makes sense!